### DISCUSSION TITLE
Crash is occurring in FreeRDP at the time of ERRINFO_LOGOFF_BY_USER in desktop screen in Android with Freerdp 2.11.2

### DISCUSSION THREAD

**Discussion by Ravi-Chandra-0708** [FreeRDP/FreeRDP#10095] [Q&A]
This issue is observed on Android , FreeRDP version 2.11.2

Entered wrong credentials in Desktop login screen & observed Username,Pwd are invalid error .
After few mins Desktop is returning ERRINFO_LOGOFF_BY_USER & leading to crash.
Observed crash was occurring due to graphics update trigger after disconnecting is called

**Comment by akallabeth** [comment]
duplicates #10101
